src/stubs.ts

```diff
@@ -112,6 +111,7 @@ export function stubComponents(
   shallow: boolean = false,
   renderStubDefaultSlot: boolean = false
 ) {
+  const component: { [key: string]: ComponentOptions } = {}
```

Comment:
You could write `Record<string, ComponentOptions>`.

Reply:
I got it !! I'll fix it soon !!